src/links.ts

```diff
@@ -45,6 +56,21 @@ export function getLinks(source: string, baseUrl: string): string[] {
 
 function parseAttr(name: string, value: string): string[] {
   switch (name) {
+    case 'style':
+      return value
+        .split(';')
+        .filter(Boolean)
```

Comment:
This feels a little magical.  Thinking it would more clear to do:
```js
.filter(line => !!line)
```

Reply:
It's the same actually. But I fix it.